tests/AnalyzerTest.elm

```diff
@@ -9,6 +9,11 @@ import Test exposing (Test, describe, test)
 import TestHelper
 
 
+tests =
```

Comment:
alright, the change looks good to me, and I see you even made it elm-test compatible by not using exactly the module name (which triggers a suprising error in elm-test)